selfdrive/locationd/calibrationd.py

```diff
@@ -46,20 +47,20 @@ class Calibration:
   INVALID = 2
 
 
-def is_calibration_valid(rpy):
+def is_calibration_valid(rpy: np.ndarray) -> bool:
```

Comment:
can we add the ndarray shape to the type annotation?

Reply:
Typing support for ndarray shapes is still a work in progress: https://github.com/numpy/numpy/issues/16544

Once that and PEP 646 support for mypy lands (https://github.com/python/mypy/issues/12280), shape annotations could be added if Python is upgraded to 3.11

Reply:
ah, cool!